### PR TITLE
Karla/homepage

### PR DESCRIPTION
**Pull request by ktechgau**
## Describe your changes

 I have begun creating the homepage 

** Please don't pay too much attention to specific styling and illustrations!!** 

This is just a rough guide so we can show Amy what we had in mind and we can deploy something Tuesday. No doubt colours,logos and anything else will be changed on Tuesday.

For all pending tasks yet to be done, please refer to Trello

NavBar: desktop view -fine, mobile view still needs some work 
Banner: Indication of what we can have there, image to be fully responsive for smaller screens.
Hero Statement: Still needs to be styled more and made more responsive.
Filter Section: This feature is yet to be made and implemented to be functional.
Profile Cards Section: Will happen once we make the profile cards!
Footer: Please refer to Trello

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## What kind of feedback do you need?

I have requested the pull feel free for anyone to continue the work needed if you like! Have a look and suggest improvements as needed.

## Checklist:

- [ x] I have performed a self-review of my code
- [x ] I have recommended where to start and how to proceed with the review

### CODE REVIEW

**Review by shayzimm — APPROVED** [review]
This is amazing, Karla. I stan 🙌

File: src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+.name{
+    width: 10rem;
+    height:auto;
+}
+
+/* NavBar Styling */
+.navbar{
+    background-color:var(--light-gray);
+    position:fixed;
```
Comment (Lyndallh) [inline src/components/NavBar/NavBar.css:23]: removing this helps the 'hidden' content appear. it is a requirement for something else in the nav like the burger menu functionality?

File: src/pages/LoginPage.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+function LoginPage() {
+  return (
+    <>
+      <h1>This is the about page</h1>
```
Comment (Lyndallh) [inline src/pages/LoginPage.jsx:4]: says about page, but directing corretly

File: src/pages/SignUpPage.jsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+function SignUpPage() {
+  return (
+    <>
+      <h1>This is the about page</h1>
```
Comment (Lyndallh) [inline src/pages/SignUpPage.jsx:4]: says 'about page', but directing correctly

**Review by Lyndallh — APPROVED** [review]
It looks good. I have left a few comments, but no action required as I can see the components are working together as expected and the things I have commented on will be addressed shortly in the next dev cycle.

File: src/components/NavBar/NavBar.css (hunk 1 above)
Comment (ktechgau) [reply src/components/NavBar/NavBar.css:23]: I'll play around with this Lindi thanks!

File: src/pages/LoginPage.jsx (hunk 2 above)
Comment (ktechgau) [reply src/pages/LoginPage.jsx:4]: will fix in the next branch thank you!

**Pull request merged**